lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -156,6 +158,10 @@
             }
 
             .course-info {
+              flex-grow: 1;
+              flex-shrink: 0;
+              min-width: 15em;
```

Comment:
Not sure what edx-platform uses, but this could be `rem` instead of `em` (relative to the root vs. relative to its nearest parent size)